src/features/organizations/components/ActivistPortlHeader/index.tsx

```diff
@@ -27,8 +30,27 @@ const ActivistPortalHeader: FC<Props> = ({
   topLeftComponent,
 }) => {
   const user = useUser();
+  const messages = useMessages(messageIds);
 
   const hasNavBar = selectedTab && tabs;
+  const [logoutMenuAnchorEl, setLogoutMenuAnchorEl] =
+    useState<HTMLButtonElement | null>(null);
+  const router = useRouter();
+  const path = usePathname();
+
+  const menuItems: MenuItem[] = [
+    {
+      label: messages.home.menu.logout(),
+      onClick: () => router.push('/logout'),
+    },
+  ];
+  if (path !== null && !path.startsWith('/my')) {
+    menuItems.unshift({
+      divider: true,
+      label: messages.home.menu.myZetkin(),
```

Comment:
Unless I'm remembering incorrectly, I believe that there is an environment variable that should be preferred here (with the `myZetkin()` message as the fallback). The purpose of the variable is to allow customization of an on-premise instance of Zetkin.